test/parallel/test-preload.js

```diff
@@ -14,14 +14,14 @@ var preloadOption = function(preloads) {
   return option;
 };
 
-var fixture = function(name) {
+const fixture = function(name) {
   return path.join(__dirname, '../fixtures/' + name);
 };
 
-var fixtureA = fixture('printA.js');
-var fixtureB = fixture('printB.js');
-var fixtureC = fixture('printC.js');
-var fixtureThrows = fixture('throws_error4.js');
+const fixtureA = fixture('printA.js');
+const fixtureB = fixture('printB.js');
+const fixtureC = fixture('printC.js');
+const fixtureThrows = fixture('throws_error4.js');
```

Comment:
I think we'd prefer if you only `const` unmodified variables you add, but if you want to do these also I really don't mind.